src/components/chat/MessageForm.js

```diff
@@ -17,54 +17,54 @@ class MessageForm extends Component {
     }
   }
 
-  onMessageUpdate(e){
+  onMessageUpdate(e) {
     this.setState({
       message: e.target.value
     });
   }
 
-  onKeyPress(e){
+  onKeyPress(e) {
     // Send on <enter> unless <shift-enter> has been pressed
-    if (e.key === 'Enter' && !e.nativeEvent.shiftKey){
+    if (e.key === 'Enter' && !e.nativeEvent.shiftKey) {
       this.onSendMessage(e);
     }
   }
 
-  isPayloadValid(message){
-    if (message && message.length > 0){
+  isPayloadValid(message) {
+    if (message && message.length > 0) {
       return true;
     }
     return false;
   }
 
-  clearMessage(){
+  clearMessage() {
     this.setState({
       message: ''
     });
   }
 
-  onSendMessage(e){
+  onSendMessage(e) {
     e.preventDefault();
 
     let { message } = this.state;
 
-    if (!this.isPayloadValid(message)){
+    if (!this.isPayloadValid(message)) {
       return false;
     }
 
     this.props.onSendMessage(message);
     this.clearMessage();
   }
 
-  render(){
+  render() {
     let { message } = this.state;
 
     return (
       <div className="message-form">
         <form role="form" className="form" onSubmit={this.onSendMessage}>
           <div>
             <Button onClick={this.onSendMessage}>
-              <i className="fa fa-arrow-circle-right fa-2x"></i>
+              <FaArrowCircleRight size={30} />
```

Comment:
It seems as though React native does not support rendering svg elements. There are several modules that solves this in different ways. But the problem is not wether components can be used both in native and in web, the problem is that components consisting of html (which is basically add components that renders something) cannot be used in React Native just like that.